src/containers/Network/index.js

```diff
@@ -38,7 +39,14 @@ class Network extends Component {
     }
     // strips :url from the front and the tab info from the end
     const base = path.split('/:')[0]
-    return tab === 'nodes' ? <Nodes path={base} /> : <Validators path={base} />
+    // eslint-disable-next-line no-nested-ternary
+    return tab === 'upgrade_status' ? (
+      <UpgradeStatus path={base} />
+    ) : tab === 'validators' ? (
+      <Validators path={base} />
+    ) : (
+      <Nodes path={base} />
+    )
```

Comment:
I'd recommend turning this into an `if/else if/else` instead of a nested ternary